packages/muelu/src/Graph/MatrixTransformation/MueLu_AmalgamationInfo_kokkos_decl.hpp

```diff
@@ -86,8 +86,11 @@ namespace MueLu {
 
   public:
 
-    AmalgamationInfo_kokkos(RCP<Array<LO> > rowTranslation,
-                            RCP<Array<LO> > colTranslation,
+    using execution_space = typename LWGraph_kokkos::execution_space;
```

Comment:
Does this need to be public?

Reply:
Maybe not but it is a type definition not a variable so I feel there is no harm in making it public?